src/applications/vaos/containers/ExpressCareInfo.jsx

```diff
@@ -20,9 +20,88 @@ class ExpressCareInfo extends React.Component {
   };
 
   render() {
+    const time = `${moment
```

Comment:
Is the difference between this and `localWindowString` that it uses "and" instead of "to"?

Reply:
Yes.